subjects/reversesecondhalf/README.md

```diff
@@ -0,0 +1,42 @@
+# ReverseSecondHalf
```

Comment:
Title should be lower case "reversesecondhalf".

Reply:
It's fixed.